ideas/100-smooth-chat-ux.md

```diff
@@ -0,0 +1,119 @@
+
+The UX issues are split between 3 groups, starting with the low effort tasks/high payoff moving to big effort/high payoff (v1, v2, v3). Although its scope can seem ambitious at first, the idea is to have deliverables once in two weeks. Those deliverables will contain all the necessary documentation necessary for implementation. More designers are welcome to join as we go, as well as developers, QA's PM's.
+
+## Swarm Participants
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
+<!-- Here all roles in swarm are defined and filled, one of the contributors should responsibility of the Idea as Lead. -->
+
+<!-- Testing/Evaluation support role is also mandatory to check in on specified Goal dates or earlier. -->
+
+<!-- Lead Contributor is the Owner of the Idea. If required, they can get support from a PM, but should be responsible for end to end execution of the Idea. This includes ensuring appropriate resources are allocated, setting realistic timelines and milestones, and any post-launch metrics or bug fixes that are attributed to the Idea -->
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: @ricardosaavedra
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+<!-- - Contributor: @username -->
+
+## Workflow and day-to-day activities
+<!-- A short (~200 word) description and motivation of the Idea. Without clear explanation the Idea should not proceed. Can include User Stories -->
+<!-- Testing/Evaluation role accepts responsbility to checkin at Goal dates, -->
+<!-- forces discussion to continue implementation or recommend disband and post-mortem. -->
+Every beginning of a new cycle (you can call it sprint if you'd like), we will decide together what to focus on for the upcoming two weeks as well as discussing current problems, technical challenges and the best solutions (trade-off between implementation cost and UX).
+As ideas and design solutions start to emerge, we will get together to validate/critique the proposed solutions until we arrive at a group consensus.
+
+
+### Product Description
+<!-- What functionality are you adding? What will this look like from a user perspective? Why is this important? -->
+
+**V1 - quick fixes**
+- Chat view / message status
+- Chat view / last seen 
+- Chat view / user page 
+- Chat view / minior UI fixes 
+- Chat view / message space optimization
+- Chat view / timestamps
+- Chat view / adding/blocking new contact flow
+- Keyboard / use cases behaviors
+- Keyboard / maximum message width
+- Keyboard / fixing console position
+- Offline states and error handling 
+
+**V2 - group chat improvements**
```

Comment:
Possibly leave this out of scope if we don't need group chat for MVP, i.e. might make more sense to have V3 before V2? Don't know - @chadyj might have an opinion